changelogs/fragments/2567-iam-user-info-handle-non-existant-group.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - iam_user_info - Get empty user list when invalid group name is provided instead of python error
```

Comment:
```suggestion
  - iam_user_info - Return empty user list when invalid group name is provided instead of python error (https://github.com/ansible-collections/amazon.aws/pull/2567).
```